explainer.md

```diff
@@ -0,0 +1,66 @@
+# Extension document explainer
+This document is a brief explainer for new features added in the [extension document](https://w3c.github.io/pointerevents/extension.html).
+
+## Coalesced Events
+Nowadays there are high frequency devices that generate user inputs much faster than frame refresh rate in the apps.
+Hence some browsers chose to not deliver all those events to the web pages as they realized it is going to be a lot of extra work that doesn't necessary have an effect on the visual output.
+For example Chromium delays all move events until right before rAF and Firefox does the same if the first event in the frame causes any changes to the DOM.
+
+### Problem
+Since most of the web apps care only about the latest state of the input before generating the next frame, the events happened during the frame could be coalesced together into one event that gets dispatched to the page.
```

Comment:
the "could be coalesced" sounds hypothetical. Instead, unless I'm misunderstanding, this is describing what is actually happening currently. So better to just say "are currently coalesced" ?